plugins/inquiry/app/mailers/inquiry/inquiry_mailer.rb

```diff
@@ -81,7 +82,17 @@ def notification_new_project(inform_dl, inquiry, user_full_name)
           subject += "/#{@inquiry.tags["domain_name"]}"
         end
       end
-      mail(to: inform_dl, subject: subject, content_type: "text/html")
```

Comment:
is that the only place where we use the mailer?

Reply:
No, we use it in several other places. This was just a single change I made to test the functionality. If we decide to proceed with this approach, we’ll need to update the other occurrences in the same file as well.